src/Analyzers/CSharp/Tests/UseImplicitObjectCreation/CSharpUseImplicitObjectCreationTests.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.UseImplicitObjectCreation;
+using Microsoft.CodeAnalysis.CSharp.UseLocalFunction;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Diagnostics;
+using Microsoft.CodeAnalysis.Editor.UnitTests.CodeActions;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.UseImplicitObjectCreationTests
+{
+    using VerifyCS = CSharpCodeFixVerifier<
+        CSharpUseImplicitObjectCreationDiagnosticAnalyzer,
+        CSharpUseImplicitObjectCreationCodeFixProvider>;
+
+    public partial class UseImplicitObjectCreationTests
```

Comment:
Partial classes don't add much with the new test framework. If you already have to copy the `using` directives to a new file it makes sense to just have a new class there.

```suggestion
    public class UseImplicitObjectCreationTests
```